### ISSUE TITLE
Issues upgrading ember cli 0.0.4 to 0.1.7

### ISSUE THREAD

**Issue by billybonks** [poetic/ember-cli-cordova#113]
i first ran these commands in order
```
rm -rf node_modules bower_components dist tmp
ember install:npm ember-cli@X.X.X
ember install
````

after running ember init i get the following error

```
Property 'config' of object #<EmberCLICordova> is not a function
TypeError: Property 'config' of object #<EmberCLICordova> is not a function
    at C:\Users\Seb\Documents\limitliss\stocks\stocks_client\node_modules\ember-
cli\lib\models\project.js:106:27
    at Array.reduce (native)
    at Project.getAddonsConfig (C:\Users\Seb\Documents\limitliss\stocks\stocks_c
lient\node_modules\ember-cli\lib\models\project.js:104:22)
    at Project.config (C:\Users\Seb\Documents\limitliss\stocks\stocks_client\nod
e_modules\ember-cli\lib\models\project.js:91:29)
    at Class.Blueprint.install (C:\Users\Seb\Documents\limitliss\stocks\stocks_c
lient\node_modules\ember-cli\lib\models\blueprint.js:331:31)
    at Class.module.exports.Task.extend.run (C:\Users\Seb\Documents\limitliss\st
ocks\stocks_client\node_modules\ember-cli\lib\tasks\install-blueprint.js:52:24)
    at Class.module.exports.Command.extend.run (C:\Users\Seb\Documents\limitliss
\stocks\stocks_client\node_modules\ember-cli\lib\commands\init.js:75:29)
    at Class.<anonymous> (C:\Users\Seb\Documents\limitliss\stocks\stocks_client\
node_modules\ember-cli\lib\models\command.js:131:17)
    at $$$internal$$tryCatch (C:\Users\Seb\Documents\limitliss\stocks\stocks_cli
ent\node_modules\ember-cli\node_modules\rsvp\dist\rsvp.js:490:16)
    at $$$internal$$invokeCallback (C:\Users\Seb\Documents\limitliss\stocks\stoc
ks_client\node_modules\ember-cli\node_modules\rsvp\dist\rsvp.js:502:17)
```

**Comment by jakecraige** [comment]
I'm not seeing this with a fresh app on 0.1.7.  I ran this series of commands:

```sh
ember new cordova-test && cd cordova-test
ember install:addon ember-cli-cordova
ember g cordova-init com.cordova.test
ember s
```

The `config` method was also changed to no longer be used in https://github.com/poetic/ember-cli-cordova/commit/5d4bd5c5e527ad4e203a03de956a6e178f34663c which was released in 0.0.15. You can try upgrading ember-cli-cordova if you haven't yet

**Comment by billybonks** [comment]
its got everything to do with how much ember-cli has changed since 0.0.4, having other issues so going to upgrade manually
